smailyforprestashop.php

```diff
@@ -251,6 +259,27 @@ public function getContent()
                 $output .= $this->displayConfirmation($this->l('Abandoned cart settings updated'));
             }
         }
+        // RSS
+        if (Tools::isSubmit('smaily_submit_rss')) {
+            $category_id = pSQL(Tools::getValue('SMAILY_RSS_CATEGORY_ID'));
+            $limit = pSQL(Tools::getValue('SMAILY_RSS_LIMIT'));
+            $order_by = pSQL(Tools::getValue('SMAILY_RSS_ORDER_BY'));
+            $order_way = pSQL(Tools::getValue('SMAILY_RSS_ORDER_WAY'));
+
+            // Check if subdomain is saved to db to verify that credentials are validated.
+            if (empty(Configuration::get('SMAILY_SUBDOMAIN'))) {
+                // Display error message.
+                $output .= $this->displayError($this->l('Please validate credentials before saving.'));
```

Comment:
Is there a reason why `SMAILY_SUBDOMAIN` needs to exist in the configuration before RSS configuration options can be saved?

Reply:
No, I realize now that RSS could be used even without an account. Removed